flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/active/ResourceManagerDriverTestBase.java

```diff
@@ -39,12 +38,10 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
 
 /** Common test cases for implementations of {@link ResourceManagerDriver}. */
-public abstract class ResourceManagerDriverTestBase<WorkerType extends ResourceIDRetrievable>
-        extends TestLogger {
+public abstract class ResourceManagerDriverTestBase<WorkerType extends ResourceIDRetrievable> {
```

Comment:
Let's annotate this test instead of adding the TestLoggerExtension service entry. As is we'd load the test logger also for all junit4 tests in flink-runtime which as far as I was told causes issues.

Reply:
Done.